cellxgene_schema_cli/setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="cellxgene-schema",
-    version="2.0.1",
+    version="2.0.0",
```

Comment:
oh, interesting, was this already incremented on `main`?

Reply:
Oh right, could use test pypi for testing this with the DP repo changes!  The official process for releasing is not buttoned down (had [started on that](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-curation/134), but never finished. In any case, following [similar process to the cellxgene desktop](https://github.com/chanzuckerberg/cellxgene/blob/main/dev_docs/release_process.md#releasing-a-major-or-minor-version-of-cellxgene) is the goal.  For test.pypi.org release, the important thing is to use release candidate version numbers, which have `-rc.#` suffixes. If 2.0.1 is not released to official pypi.org yet, I would use `2.0.1-rc.1`, and if 2.01 is officially release, I would bump to `2.0.2-rc.1`.